idea/src/org/jetbrains/kotlin/idea/refactoring/changeSignature/usages/KotlinFunctionCallUsage.kt

```diff
@@ -101,7 +99,10 @@ class KotlinFunctionCallUsage(
         if (resolvedCall == null || resolvedCall.isReallySuccess()) return false
 
         // TODO: investigate why arguments are not recorded for enum constructor call
-        if (element is KtSuperTypeCallEntry && element.parent.parent is KtEnumEntry) return false
+        val enumEntry = element.parent.parent as? KtEnumEntry
+        if (element is KtSuperTypeCallEntry && enumEntry != null) {
+            if (element.valueArguments.isEmpty() && enumEntry.containingClass()?.constructor?.valueParameters?.size == 1) return false
```

Comment:
I suppose this check covers only one particular case. See also test comment below.